Packs/FortiAuthenticator/Integrations/FortiAuthenticator/README.md

```diff
@@ -0,0 +1,114 @@
+
+FortiAuthenticator provides centralized authentication services. 
+Seamless secure two-factor/OTP authentication across the organization in conjunction with FortiToken.

```

Comment:
can we add here the version of FortiAuthenticator this integration was tested against ?

Reply:
Agreed. The supported FortiAuthenticator versions:  4.0.x - 6.3.x